clients/admin-ui/src/features/system/add-multiple-systems/MultipleSystemsFilterModal.tsx

```diff
@@ -0,0 +1,150 @@
+import {
+  Box,
+  Button,
+  Checkbox,
+  Divider,
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+  Text,
+} from "@fidesui/react";
+import { Table as TableInstance } from "@tanstack/react-table";
+import React, { useMemo, useState } from "react";
+
+type FilterCheckboxProps = {
+  onChange: () => void;
+  displayText: string;
+  value: string;
+  isChecked: boolean;
+};
+
+const FilterCheckbox = ({
+  onChange,
+  displayText,
+  isChecked,
+  value,
+}: FilterCheckboxProps) => (
+  <Checkbox
+    value={value}
+    key={value}
+    height="20px"
+    mb={3}
+    isChecked={isChecked}
+    onChange={onChange}
+    colorScheme="complimentary"
+    mr={5}
+  >
+    <Text
+      fontSize="sm"
+      lineHeight={5}
+      height="20px"
+      textOverflow="ellipsis"
+      overflow="hidden"
+      whiteSpace="nowrap"
+    >
+      {displayText}
+    </Text>
+  </Checkbox>
+);
+
+interface MultipleSystemsFilterProps<T> {
+  tableInstance: TableInstance<T>;
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const initialFilterState = {
+  gvl: false,
+  gacp: false,
```

Comment:
shouldn't this be `true` for both? at the beginning, both GVL and AC are shown in sources, but neither of the check boxes are checked

```suggestion
  gvl: true,
  gacp: true,
```

![image](https://github.com/ethyca/fides/assets/24641006/653b0b58-fbc6-4d8b-92aa-bca9c99f01ca)

Reply:
AFAIK this is actually the desired behavior and it's how the datamap currently works.

![Screenshot from 2023-10-31 11-25-37](https://github.com/ethyca/fides/assets/17103888/702e872d-9067-4dbe-b84e-363695f7d737)



I don't remember the exact conversation that Rachel and I had about a year ago for the map but we ended up there. It's not as obvious when there are only two checkboxes but on the modal above where there are many it can cause issues. It changes the using the modal from `check the one, exact, option I want to look at` to `uncheck every single checkbox that isn't the one thing I want to look at`. The first option allows the user to additivly and explicitly build theirs filters up. Essentially, it's an odd use case to "filter" the table and display everything.


From a code perspective there aren't any filters yet either. There is a list of filter objects that get appended to when those checkboxes are checked. When none are checked the list is empty,

I think it should stay how it currently is to match how the datamap works and we can revisit it in the future we we decide the subtractive fitler method is better

Reply:
okay, thanks for the context!